config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -152,4 +152,17 @@ endif # SOC_SERIES_NRF53X
 
 endif # CHIP_DFU_OVER_BT_SMP
 
+if MPSL_FEM
+
+config MPSL_FEM_NRF21540_RUNTIME_PA_GAIN_CONTROL
+	bool
+	default y
+
+# Set 10 dBm output power for FEM devices as default
+config OPENTHREAD_DEFAULT_TX_POWER
+	int
+	default 10
```

Comment:
I am fine with setting the output power to 10dBm by default. But, we should keep it to 0dBm for SED devices.
Also, please document in the release notes (and somewhere in the documentation) what is the exact value of output power for: `nrf52840` (FTD, (S)SED) and `nRF5340` (FTD, (S)SED).